modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
```

Comment:
If this has a title like "Configuring" and shows how to configure settings it should probably be a procedure with numbered steps instead of a `CONCEPT` module.

Reply:
How does this differ from `logging-loki-restart-hardening.adoc` ?

Reply:
That doc just shows an example of how to configure a setting in one file, this one shows settings for LokiStack CR plus different role bindings, so it seems like those things need to be configured together to make this work? Maybe not, though.

Reply:
There is no action the user needs to take outlined here. The examples are exposures of default values requiring no configuration by the user. This is a 'hey this is a new setting, here's where it is & what the fields mean' module not a 'how to do specific use-case xyz'.